stocklist.py

```diff
@@ -2,6 +2,10 @@
 import os
 import errno
 
+
+# this is used to get all tickers from the market.
+
+
 exportList = []
```

Comment:
There was a pull request which put the export list in a class and made it a return of the script. I think that's a very good idea, because like my other comment, allows for greater flexibility.